nodebuilder/p2p/flags.go

```diff
@@ -2,28 +2,41 @@ package p2p
 
 import (
 	"fmt"
+	"os"
+	"strings"
 
 	"github.com/multiformats/go-multiaddr"
 	"github.com/spf13/cobra"
 	flag "github.com/spf13/pflag"
 )
 
-var (
-	p2pMutualFlag = "p2p.mutual"
+const (
+	// DefaultNetwork is the default network of the current build.
+	DefaultNetwork   = Arabica
+	EnvCustomNetwork = "CELESTIA_CUSTOM"
+	networkFlag      = "p2p.network"
+	mutualFlag       = "p2p.mutual"
 )
```

Comment:
optional nit: all the const does not have to be under one const keyword group. In this case, we could clearly separate flag names from Network related costs. Further, we could move DefaultNetwork const closer to where the Network is defined